fastlane/report.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
chore: Please ignore the report from git

Reply:
@timo-reymann thank you for your quick review and suggestion. This PR was just for testing working status of the DCO git app, therefore I am deleting it. 

In the new PR, I won't include the fastlane reports xml for sure 👍